### DISCUSSION TITLE
Problem Interstitial if internet is offline

### DISCUSSION THREAD

**Discussion by RayosElDev** [invertase/react-native-firebase#5191] [General]
Hello
If I try to do an 
  interstitialAd.load();
 when is Offline , is normal that No work.

But when Internet back,

Interstitial No Load, and if I try to call
  interstitialAd.load();

still no load.

Why?
Thanks

**Comment by mikehardy** [comment]
This was just covered recently, you may like: https://github.com/invertase/react-native-firebase/issues/5093#issuecomment-821858873

That comment contains the code I used literally yesterday to troubleshoot a similar problem, and with the code (from our documentation) it worked for me. With the code from the user having problems I was *not* able to load Interstitial ads so it's clear the code is touch-y, and the comment contains code that works.

Hope this helps
